pkg/config/setup/system_probe_cws.go

```diff
@@ -132,4 +132,10 @@ func initCWSSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	cfg.BindEnvAndSetDefault("runtime_security_config.enforcement.enabled", true)
 	cfg.BindEnvAndSetDefault("runtime_security_config.enforcement.raw_syscall.enabled", false)
 	cfg.BindEnvAndSetDefault("runtime_security_config.enforcement.exclude_binaries", []string{})
+	cfg.BindEnvAndSetDefault("runtime_security_config.enforcement.disarmer.container.enabled", false)
```

Comment:
I would enabled it by default

Reply:
Done with commit ac703dbb2e75d0e158a0edda94fde72d6692ebd7